src/schemas/variant.json

```diff
@@ -49,6 +49,20 @@
           "required": {
             "type": "boolean",
             "description": "Boolean indicating whether or not the component is required"
+          },
+          "dependency": {
+            "type": "array",
+            "description": "Array containing list of all components from which depends current conponent",
+            "items": {
+              "type": "string"
+            }
+          },
+          "subtheme": {
+            "type": "array",
+            "description": "List of all subtheme to which this component belonning to and should be installed as part of subtheme",
+            "items": {
+              "type": "string"
+            }
```

Comment:
What is the `subtheme` property for? I didn't see an example of its use in here, and I couldn't determine from the description what the term "theme" means in this context. Can you clarify that?

Reply:
Thanks for your review.
The compound theme could contain styles for different websites which is in here related to the subtheme. Front end development team could prepare and build components in the theme's own repo before the back-end team could use it on the website. Once components are ready to use backend development team could pull components not based on one component at a time, but by synchronizing the full subtheme. The exact process will be during deployment. emulsify-cli tool could prepare deployment artifacts in the same way as composer pulling all components from central repo based on subtheme.
I didn't finish development for the subtheme which is why you see only configuration without code. If you agree that this could be a valuable add-on I will add a code update with an additional option in `emulsify component install -subtheme <name>`

Reply:
I think I might understand, but it would be good to see that fleshed out more in a separate PR. Would you mind pulling the subtheme stuff out of this one, so that this PR is focused on the dependency feature, and then submitting a separate PR for the subtheme idea?

I have a couple of our typescript developers queued to do a code review of this PR. I'd love to get it in!

Reply:
Subtheme structure removed. Dependency works together with compound theme changes 
https://github.com/emulsify-ds/compound/pull/44/files